src/classes/CallableApiApexDelegationService.cls

```diff
@@ -0,0 +1,66 @@
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2021
+* @group API
+* @group-content ../../ApexDocContent/API.htm
+* @description API class that handles apex jobs
+*/
+
+public with sharing class CallableApiApexDelegationService {
+
+    /*******************************************************************************************************
+    * @description Handles scheduling apex jobs specified in params
+    * @param params name value map of arguments
+    * @return Object returned is String with CronTrigger.Id
+    ********************************************************************************************************/
+    public static Object handleApexScheduleJob(Map<String, Object> params) {
+        // This method schedules the specified Schedulable class using the specified job name & cron expression.
+        // It should only be used for Schedulable classes that are in the NPSP Managed Package.
+        // It returns the Job ID of the Scheduled Job.
+
+        CallableApiParameters paramService = new CallableApiParameters(params);
+
+        String jobName = paramService.getString(CallableApiParameters.PARAM_SCHEDULED_JOB_NAME);
+        String cronExpression = paramService.getString(CallableApiParameters.PARAM_SCHEDULED_JOB_CRON_EXPRESSION);
+        String apexClass = paramService.getString(CallableApiParameters.PARAM_SCHEDULED_JOB_CLASS);
+
+        Type apexClassType = Type.forName(apexClass);
+        if (apexClassType == null) {
+            throw new Callable_API.MalformedMethodInvocationException(
+                    String.format(System.Label.CallableApiJobScheduleInvalidClass, new List<String> { apexClass })
+            );
+        }
+
+        Schedulable schedulableClass = (Schedulable) apexClassType.newInstance();
```

Comment:
It might be good to wrap this in a try catch block as well in case the cron expression or job name causes a failure - unless the expectation is that the pass through exception is valid - which it might be. Alternatively we could put the cron expression through a [regex](https://stackoverflow.com/questions/14203122/create-a-regular-expression-for-cron-statement/19046939) to validate it.

Reply:
An invalid cron expression causes an apex error that is already clear than an invalid cron expression was the issue.  Do you think there is value in throwing our own error?

Reply:
That's fine then. Mostly just making sure that's accounted for. We shouldn't do this now, but it would be interesting if we could support cron expressions like "IN_MINUTES:N" where it parses out the minutes value and dynamically generates a cron expression to run the job once _N_ minutes from the current time. We can wait for a customer to request that kind of functionality.

Reply:
Yes, that would be nice.  Looking forward to extending and improving this functionality!